pgraph/pgraph_test.go

```diff
@@ -883,4 +883,37 @@ func TestFindEdge1(t *testing.T) {
 	if g.FindEdge(v1, v7) != nil {
 		t.Errorf("an edge was found although it did not exist")
 	}
+
```

Comment:
No need to add this extra newline.

Reply:
I removed the line and edited the message.